cmd/maya-agent/app/block_devices_test.go

```diff
@@ -0,0 +1,23 @@
+package app_test
+
+import (
+	"fmt"
+	"os/exec"
+	"testing"
+)
+
+func TestListBlockDevice(t *testing.T) {
+	res, err := exec.Command("../maya-agent", "block", "list").Output()
```

Comment:
This unit testing expects the presence of maya-agent binary.

Q1/ How to ensure the latest binary is available before this test command is executed ?

Reply:
Since maya-agent is a separate binary and the make file right now is not modified to build and test maya-agent, I have given a customized procedure to build and test the same, on the pull request. Kiran will be working the makefile and shell script to make it compatible with maya-agent.

Reply:
@gkGaneshR Thanks for explaining.
Keep a check on the unit testing process vs. e2e based test process.
The format you have been trying here is best for e2e.

The unit test should always be scoped to a function without any dependencies e.g. the mayactl binary is a dependency here.